README.rst

```diff
@@ -8,48 +8,52 @@ Please tag **@edx/arch-team** on any PRs or issues.  Thanks.
 Introduction
 ------------
 
-Micro-frontend for the single-page payment/checkout process. This application supports Apple Pay, PayPal, and Cybersource credit card payment types.
+Micro-frontend for the single-page payment/checkout process; order history and receipt pages are still served by the ecommerce service. This application only supports PayPal and Cybersource credit card payments.
 
 Getting Started
 ---------------
 
-**Prerequisite**
+This MFE is bundled with `Devstack <https://github.com/edx/devstack>`_, see the `Getting Started <https://github.com/edx/devstack#getting-started>`_ section for setup instructions.
 
-`Devstack <https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/installation/index.html>`_.  If you start Devstack with ``make dev.up.ecommerce`` that should give you everything you need as a companion to this frontend.
+Once set up, this frontend and all its prerequisites can be started with:
 
-**Installation and Startup**
+.. code-block::
 
-1. Clone this repo:
+  cd devstack
+  make dev.up.frontend-app-payment
 
-  ``git clone https://github.com/edx/frontend-app-payment.git``
+The frontend runs at `http://localhost:1998 <http://localhost:1998>`_, though it is rare to interact with it directly.  Rather, log into the LMS at http://localhost:18000/login and enroll in a course, choosing to upgrade to be taken to the checkout page.
```

Comment:
Maybe add some reasoning to explain why it's rare to interact with it directly? e.g. "it is rare to interact with the url directly because the course key needs to be passed as a URL param to determine basket contents to check out"

Reply:
Good thought, added.